source/_integrations/keyboard_remote.markdown

```diff
@@ -165,3 +165,29 @@ You can check ACLs permissions with:
 ```bash
 getfacl /dev/input/event*
 ```
+
+## Containers
+
+If you are running Home Assistant Container, you need to pass the input device through to the container. This can be done by running the container in privileged mode, but this is is not ideal for security. Instead you can pass the input device you want to use directly into the container with the `--devices` flag. However, restarting the container or unplugging and replugging your keyboard will break this integration as only the instance of the keyboard that existed when the container first starts will be avaible inside of the container.
```

Comment:
```suggestion
If you are running Home Assistant Container, you need to pass the input device through to the container. This can be done by running the container in privileged mode, but this is not ideal for security. Instead, you can pass the input device you want to use directly into the container with the `--devices` flag. However, restarting the container or unplugging and replugging your keyboard will break this integration. This is because only the instance of the keyboard that existed when the container first started will be available inside the container.
```